packages/block-library/src/block/editor.scss

```diff
@@ -18,7 +18,7 @@
 // Styles for save button
 .components-toolbar-group {
 	.block-library-block__reusable-block-save-button {
-		margin: 7px;
+		margin: 8px;
```

Comment:
You can use $grid-unit-10 instead of 8px if you like.

Reply:
Oh, Yes! Thank you! 😄